RxCocoa/iOS/UIDatePicker+Rx.swift

```diff
@@ -30,7 +35,18 @@ extension Reactive where Base: UIDatePicker {
             }
         )
     }
-    
+
+    /// Reactive wrapper for `countDownDuration` property.
+    public var countDownValue: ControlProperty<TimeInterval> {
```

Comment:
Hi @pawurb ,

I'm not sure what would be the proper naming conventions for this case. Let's only add `countDownDuration` property for now and just move this implementation there.